Global.asax.cs

```diff
@@ -10,7 +10,7 @@ public class Global : System.Web.HttpApplication
         protected void Application_Start()
         {
             Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration
-                .Active.InstrumentationKey = ConfigurationManager.AppSettings["AppInsightKey"] ?? System.Environment.GetEnvironmentVariable("AppInsightKey");
+                .Active.InstrumentationKey = ConfigurationManager.AppSettings["AppInsightKey"] ?? System.Environment.GetEnvironmentVariable("AppInsightKey") ?? string.Empty;
```

Comment:
Why does it need to fall back to env var, since Azure AppSettings, end up properly populating `ConfigurationManager.AppSettings`?